### DISCUSSION TITLE
manual refresh rate acts like auto

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#1689] [Q&A]
Hey,

I would like to keep my computer at 60hz all the time (plugged and unplugged). 
I set the refresh rate in Ghelper at 60hz while plugged. If I unplug and replug the computer, it goes back to 165Hz, Even tho I set it to 60hz.

I tried a DDU and then reinstall the AMD and Nvidia drivers (from the Ghelper update links). It didn't change anything. I also got rid of Nvidia and AMD control panels. The problem is still there. 

Any help would be welcome. Thanks a lot !

 Zephyrus G15 2021 GA503QE bios 416

**Comment by seerge** [comment]
@romainbtl hello, if it's set to 60Hz in the app, it won't even try to change your refresh rate. Since you didn't include logs, i can't tell more. 

Just make sure that you have uninstalled Asus Smart Display, as it's switching refresh rates as well
https://github.com/seerge/g-helper#recommendations-optional
